From b8154f1fba5496be524fb110c78ffe49ec58d3f1 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Mon, 4 Jun 2018 09:35:12 +0800
Subject: [PATCH] Revert "arm64 Kconfig: Move LIST_POISON to a safe value"

This reverts commit 4003f6b715791eddb3eb429fb3787236403d320a.

This is a redefine of commit bf0c4e047 (arm64: kconfig: Move LIST_POISON
to a safe value)

Change-Id: Ibde683271af2bc982f3727a98ebd1530fc503dd1
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/Kconfig | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index 22364fcf5eb3..2cbb42f7366a 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -158,10 +158,6 @@ config ARM64_CONT_SHIFT
 config NO_IOPORT_MAP
 	def_bool y if !PCI
 
-config ILLEGAL_POINTER_VALUE
-	hex
-	default 0xdead000000000000
-
 config STACKTRACE_SUPPORT
 	def_bool y
 
-- 
2.35.3

